client/src/components/Telepath/Telepath.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable dot-notation, no-param-reassign */
```

Comment:
I think this belongs in either `utils` or its own root folder rather than in components